ERROR    root:test_LastConfigCompleted.py:97 ❌ Error in configurator: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#cc-app-container-main > div.cc-app-container__main-frame.cc-grid-container > div.cc-app-container__navigation.ng-star-inserted > cc-navigation > nav > div > ul > li:nth-child(3) > ccwb-text > a"}
  (Session info: chrome=135.0.7049.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
#0 0x555806d9b75a <unknown>
#1 0x55580684e4b0 <unknown>
#2 0x55580689f9b3 <unknown>
#3 0x55580689fbd1 <unknown>
#4 0x5558068934b6 <unknown>
#5 0x5558068c55bd <unknown>
#6 0x55580689336a <unknown>
#7 0x5558068c575e <unknown>
#8 0x5558068eb5e0 <unknown>
#9 0x5558068c5363 <unknown>
#10 0x555806891d63 <unknown>
#11 0x5558068929c1 <unknown>
#12 0x555806d60a6b <unknown>
#13 0x555806d64951 <unknown>
#14 0x555806d48b62 <unknown>
#15 0x555806d654c4 <unknown>
#16 0x555806d2d13f <unknown>
#17 0x555806d896f8 <unknown>
#18 0x555806d898d6 <unknown>
#19 0x555806d9a5a6 <unknown>
#20 0x7eff51a9caa4 <unknown>
#21 0x7eff51b29c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:87 ❌ Image not found in the specified selector. Error: Message: 

ERROR    root:VerifyPersonalizationAndCapture.py:18 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Perso/QA-Perso/Tests/Last Configuration Completed_attempt_2.png'
ERROR    root:QAAppAllure.py:172 ❌ Test 'Last Configuration Completed' failed.
ERROR    root:test_LastConfigCompleted.py:97 ❌ Error in configurator: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#cc-app-container-main > div.cc-app-container__main-frame.cc-grid-container > div.cc-app-container__navigation.ng-star-inserted > cc-navigation > nav > div > ul > li:nth-child(3) > ccwb-text > a"}
  (Session info: chrome=135.0.7049.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
#0 0x558862dba75a <unknown>
#1 0x55886286d4b0 <unknown>
#2 0x5588628be9b3 <unknown>
#3 0x5588628bebd1 <unknown>
#4 0x5588628b24b6 <unknown>
#5 0x5588628e45bd <unknown>
#6 0x5588628b236a <unknown>
#7 0x5588628e475e <unknown>
#8 0x55886290a5e0 <unknown>
#9 0x5588628e4363 <unknown>
#10 0x5588628b0d63 <unknown>
#11 0x5588628b19c1 <unknown>
#12 0x558862d7fa6b <unknown>
#13 0x558862d83951 <unknown>
#14 0x558862d67b62 <unknown>
#15 0x558862d844c4 <unknown>
#16 0x558862d4c13f <unknown>
#17 0x558862da86f8 <unknown>
#18 0x558862da88d6 <unknown>
#19 0x558862db95a6 <unknown>
#20 0x7fac1f69caa4 <unknown>
#21 0x7fac1f729c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:87 ❌ Image not found in the specified selector. Error: Message: 

ERROR    root:VerifyPersonalizationAndCapture.py:18 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Perso/QA-Perso/Tests/Last Configuration Completed_attempt_2.png'
ERROR    root:QAAppAllure.py:172 ❌ Test 'Last Configuration Completed' failed.
ERROR    root:test_LastConfigCompleted.py:97 ❌ Error in configurator: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#cc-app-container-main > div.cc-app-container__main-frame.cc-grid-container > div.cc-app-container__navigation.ng-star-inserted > cc-navigation > nav > div > ul > li:nth-child(3) > ccwb-text > a"}
  (Session info: chrome=135.0.7049.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
#0 0x563af59c375a <unknown>
#1 0x563af54764b0 <unknown>
#2 0x563af54c79b3 <unknown>
#3 0x563af54c7bd1 <unknown>
#4 0x563af54bb4b6 <unknown>
#5 0x563af54ed5bd <unknown>
#6 0x563af54bb36a <unknown>
#7 0x563af54ed75e <unknown>
#8 0x563af55135e0 <unknown>
#9 0x563af54ed363 <unknown>
#10 0x563af54b9d63 <unknown>
#11 0x563af54ba9c1 <unknown>
#12 0x563af5988a6b <unknown>
#13 0x563af598c951 <unknown>
#14 0x563af5970b62 <unknown>
#15 0x563af598d4c4 <unknown>
#16 0x563af595513f <unknown>
#17 0x563af59b16f8 <unknown>
#18 0x563af59b18d6 <unknown>
#19 0x563af59c25a6 <unknown>
#20 0x7efe2649caa4 <unknown>
#21 0x7efe26529c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:87 ❌ Image not found in the specified selector. Error: Message: 

ERROR    root:VerifyPersonalizationAndCapture.py:18 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Perso/QA-Perso/Tests/Last Configuration Completed_attempt_2.png'
ERROR    root:QAAppAllure.py:172 ❌ Test 'Last Configuration Completed' failed.
ERROR    root:test_LastConfigCompleted.py:97 ❌ Error in configurator: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#cc-app-container-main > div.cc-app-container__main-frame.cc-grid-container > div.cc-app-container__navigation.ng-star-inserted > cc-navigation > nav > div > ul > li:nth-child(3) > ccwb-text > a"}
  (Session info: chrome=135.0.7049.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
#0 0x55d4f7aa775a <unknown>
#1 0x55d4f755a4b0 <unknown>
#2 0x55d4f75ab9b3 <unknown>
#3 0x55d4f75abbd1 <unknown>
#4 0x55d4f759f4b6 <unknown>
#5 0x55d4f75d15bd <unknown>
#6 0x55d4f759f36a <unknown>
#7 0x55d4f75d175e <unknown>
#8 0x55d4f75f75e0 <unknown>
#9 0x55d4f75d1363 <unknown>
#10 0x55d4f759dd63 <unknown>
#11 0x55d4f759e9c1 <unknown>
#12 0x55d4f7a6ca6b <unknown>
#13 0x55d4f7a70951 <unknown>
#14 0x55d4f7a54b62 <unknown>
#15 0x55d4f7a714c4 <unknown>
#16 0x55d4f7a3913f <unknown>
#17 0x55d4f7a956f8 <unknown>
#18 0x55d4f7a958d6 <unknown>
#19 0x55d4f7aa65a6 <unknown>
#20 0x7f503529caa4 <unknown>
#21 0x7f5035329c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:87 ❌ Image not found in the specified selector. Error: Message: 

ERROR    root:VerifyPersonalizationAndCapture.py:18 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Perso/QA-Perso/Tests/Last Configuration Completed_attempt_2.png'
ERROR    root:QAAppAllure.py:172 ❌ Test 'Last Configuration Completed' failed.
ERROR    root:test_LastConfigCompleted.py:97 ❌ Error in configurator: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#cc-app-container-main > div.cc-app-container__main-frame.cc-grid-container > div.cc-app-container__navigation.ng-star-inserted > cc-navigation > nav > div > ul > li:nth-child(3) > ccwb-text > a"}
  (Session info: chrome=135.0.7049.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
#0 0x55c676c5675a <unknown>
#1 0x55c6767094b0 <unknown>
#2 0x55c67675a9b3 <unknown>
#3 0x55c67675abd1 <unknown>
#4 0x55c67674e4b6 <unknown>
#5 0x55c6767805bd <unknown>
#6 0x55c67674e36a <unknown>
#7 0x55c67678075e <unknown>
#8 0x55c6767a65e0 <unknown>
#9 0x55c676780363 <unknown>
#10 0x55c67674cd63 <unknown>
#11 0x55c67674d9c1 <unknown>
#12 0x55c676c1ba6b <unknown>
#13 0x55c676c1f951 <unknown>
#14 0x55c676c03b62 <unknown>
#15 0x55c676c204c4 <unknown>
#16 0x55c676be813f <unknown>
#17 0x55c676c446f8 <unknown>
#18 0x55c676c448d6 <unknown>
#19 0x55c676c555a6 <unknown>
#20 0x7f6a0f69caa4 <unknown>
#21 0x7f6a0f729c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:87 ❌ Image not found in the specified selector. Error: Message: 

ERROR    root:VerifyPersonalizationAndCapture.py:18 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Perso/QA-Perso/Tests/Last Configuration Completed_attempt_2.png'
ERROR    root:QAAppAllure.py:172 ❌ Test 'Last Configuration Completed' failed.
ERROR    root:test_LastConfigCompleted.py:97 ❌ Error in configurator: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#cc-app-container-main > div.cc-app-container__main-frame.cc-grid-container > div.cc-app-container__navigation.ng-star-inserted > cc-navigation > nav > div > ul > li:nth-child(3) > ccwb-text > a"}
  (Session info: chrome=135.0.7049.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
#0 0x55bba356475a <unknown>
#1 0x55bba30174b0 <unknown>
#2 0x55bba30689b3 <unknown>
#3 0x55bba3068bd1 <unknown>
#4 0x55bba305c4b6 <unknown>
#5 0x55bba308e5bd <unknown>
#6 0x55bba305c36a <unknown>
#7 0x55bba308e75e <unknown>
#8 0x55bba30b45e0 <unknown>
#9 0x55bba308e363 <unknown>
#10 0x55bba305ad63 <unknown>
#11 0x55bba305b9c1 <unknown>
#12 0x55bba3529a6b <unknown>
#13 0x55bba352d951 <unknown>
#14 0x55bba3511b62 <unknown>
#15 0x55bba352e4c4 <unknown>
#16 0x55bba34f613f <unknown>
#17 0x55bba35526f8 <unknown>
#18 0x55bba35528d6 <unknown>
#19 0x55bba35635a6 <unknown>
#20 0x7f617fe9caa4 <unknown>
#21 0x7f617ff29c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:87 ❌ Image not found in the specified selector. Error: Message: 

ERROR    root:VerifyPersonalizationAndCapture.py:18 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Perso/QA-Perso/Tests/Last Configuration Completed_attempt_2.png'
ERROR    root:QAAppAllure.py:172 ❌ Test 'Last Configuration Completed' failed.